src/components/common/Pagination/Pagination.test.tsx

```diff
@@ -0,0 +1,96 @@
+import { render, screen } from "@testing-library/react";
+import { expect } from "vitest";
```

Comment:
remove this import skeby

Reply:
I don't know what you mean by "vitest globals", but when I removed the imports, I was seeing errors and the tests didn't run anymore